.dockerignore

```diff
@@ -3,5 +3,7 @@
 .idea
 build
 docs
+provisioning
```

Comment:
To je asi typo, ne? 🤔

Reply:
adresar sa tak vola, ci nie?
![image](https://user-images.githubusercontent.com/19371734/198968619-b693d0f3-510e-4703-9835-0d54ce9fabb2.png)

alebo nevidim dobre :slightly_smiling_face:  kde je preklep?

Reply:
omg sorry, já myslel že je to `.gitignore` 🙂